.gitignore

```diff
@@ -10,6 +10,10 @@ defaults/colors.tsv
 defaults/colors_*.tsv
 stats.json
 
+# Downloaded remote files from sources we expect
+/nextstrain-data/
+/data.nextstrain.org/
```

Comment:
I wish these remote data directories could be easily nested in the existing `data/` directory, to follow the pattern we've been demonstrating to users at workshops and in the documentation. I see though that this is a Snakemake issue and not an issue with this PR.

Reply:
Same. I looked into it, and to do so is technically possible, but involves either a lot of monkey-patching Snakemake or writing our own multi-protocol `snakemake.remote.AbstractRemoteProvider` implementation class that supports HTTPS, S3, GS, etc without proxying to the existing single-protocol implementations. I just don't think it is worth it.

Reply:
No more monkeys...

Reply:
…jumping in the code.